src/Microsoft.TestPlatform.ObjectModel/TestResult.cs

```diff
@@ -36,8 +36,8 @@ public TestResult(TestCase testCase)
 
             // Default start and end time values for a test result are initialized to current timestamp
             // to maintain compatibility.
-            this.StartTime = DateTimeOffset.Now;
-            this.EndTime = DateTimeOffset.Now;
+            this.StartTime = DateTimeOffset.UtcNow;
```

Comment:
This could change the behavior for inner loop scenarios. If any client displaying time it's client responsibility to localize the date. TestPlatform will use UTC for consistence.

Reply:
As we discussed, it might be. But we need to draw this and follow everywhere as UTC standard.

Reply:
Yes, Totally agreed.